### PR TITLE
FTP: Fixed viewmodel issues (72176, 72177, 72178)

### PR DESCRIPTION
**Pull request by alexandru-petre**
Enabled view models on Windows projects.
Customized the data source of the `SSLProtocols` property from the `UseFTPConnection` activity. (STUD-72176)
Fixed some resource strings.
Fixed the widgets of some boolean properties. (STUD-72177)
Cleaned the code of view models and fixed their appearance in StudioWeb. (STUD-72178)
Touched the solution files of several packages because the shared projects were not included in their solutions

https://uipath.atlassian.net/browse/STUD-72176
https://uipath.atlassian.net/browse/STUD-72177
https://uipath.atlassian.net/browse/STUD-72178

### CODE REVIEW

File: Activities/Shared/UiPath.Shared/SystemExtensions.cs [hunk 1]
```diff
@@ -19,9 +19,9 @@ public static IComparer<T> ToComparer<T>(this Func<T, T, int> compare)
             return new Comparer<T>(compare);
         }
 
-        private class Comparer<T> : IComparer<T>
+        private sealed class Comparer<T> : IComparer<T>
```
Comment (viogroza) [inline Activities/Shared/UiPath.Shared/SystemExtensions.cs:22]: does the Extensions class need to be made internal also since is in a shared project?

File: Activities/Shared/UiPath.Shared.Activities/EnumExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.CodeDom;
```
Comment (viogroza) [inline Activities/Shared/UiPath.Shared.Activities/EnumExtensions.cs]: unused namespaces

File: Activities/Shared/UiPath.Shared.Activities/EnumExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+using System.Linq;
+using System.Text;
+
+namespace UiPath.Shared.Activities
+{
+    internal static class EnumExtensions<T> where T : struct
+    {
+        public static bool EnumTryParseNoIntegers(string value, out T result)
+        {
+            if (int.TryParse(value, out _))
```
Comment (viogroza) [inline Activities/Shared/UiPath.Shared.Activities/EnumExtensions.cs]: why return default if we succeed to parse here?

File: Activities/Shared/UiPath.Shared.Activities/EnumExtensions.cs (hunk 2 above)
Comment (alexandru-petre) [reply Activities/Shared/UiPath.Shared.Activities/EnumExtensions.cs]: fixed

File: Activities/Shared/UiPath.Shared.Activities/EnumExtensions.cs (hunk 3 above)
Comment (alexandru-petre) [reply Activities/Shared/UiPath.Shared.Activities/EnumExtensions.cs]: this code was trying to eliminate integer values and allow only enum names, because Enum.TryParse will return true for non-existent integer values (it will return that value as if it's part of the enum)

anyway, the method was copied from system but it was useless so I've dropped it

File: Activities/Activities.Python.sln [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30621.155
```
Comment (viogroza) [inline Activities/Activities.Python.sln:4]: this change (and a few other) doesn't seem related to the jira tickets that are linked to this ticket

**Review by viogroza — APPROVED** [review]

File: Activities/Activities.Python.sln (hunk 4 above)
Comment (alexandru-petre) [reply Activities/Activities.Python.sln:4]: source files from the shared projects were used but not included in the solution
